schemas/components/schemas/Visit.yaml

```
@@ -1,110 +1,59 @@
 type: object
 additionalProperties: false
+required:
+  - requestId
+  - browserDetails
+  - incognito
+  - ip
+  - timestamp
+  - time
+  - url
+  - tag
+  - visitorFound
+  - firstSeenAt
+  - lastSeenAt
 properties:
   requestId:
-    description: Unique identifier of the user's identification request.
     type: string
-    example: 1654815516083.OX6kx8
+    description: Unique identifier of the user's request.
   browserDetails:
     $ref: BrowserDetails.yaml
   incognito:
-    description: Flag if user used incognito session.
     type: boolean
+    description: Flag if user used incognito session.
   ip:
     type: string
     format: ipv4
-    example: 8.8.8.8
+    description: IP address of the requesting browser or bot.
   ipLocation:
-    type: object
-    additionalProperties: false
-    deprecated: true
-    description: >-
-      This field is **deprecated** and will not return a result for
-      **applications created after January 23rd, 2024**. Please use the [IP
-      Geolocation Smart
-      signal](https://dev.fingerprint.com/docs/smart-signals-overview#ip-geolocation)
-      for geolocation information.
-    properties:
-      accuracyRadius:
-        description: >-
-          The IP address is likely to be within this radius (in km) of the
-          specified location.
-        type: integer
-        minimum: 0
-      latitude:
-        type: number
-        format: double
-        minimum: -90
-        maximum: 90
-      longitude:
-        type: number
-        format: double
-        minimum: -180
-        maximum: 180
-      postalCode:
-        type: string
-      timezone:
-        type: string
-        format: timezone
-      city:
-        type: object
-        additionalProperties: false
-        title: DeprecatedIPLocationCity
-        properties:
-          name:
-            type: string
-      country:
-        $ref: Location.yaml
-      continent:
-        $ref: Location.yaml
-      subdivisions:
-        type: array
-        items:
-          $ref: Subdivision.yaml
-    title: DeprecatedIPLocation
+    $ref: DeprecatedGeolocation.yaml
+  linkedId:
+    type: string
+    description: A customer-provided id that was sent with the request.
   timestamp:
-    description: Timestamp of the event with millisecond precision in Unix time.
     type: integer
     format: int64
-    example: 1654815516086
+    description: Timestamp of the event with millisecond precision in Unix time.
   time:
-    title: Time
-    description: Time expressed according to ISO 8601 in UTC format.
     type: string
     format: date-time
-    example: '2022-06-09T22:58:36Z'
+    description: >-
+      Time expressed according to ISO 8601 in UTC format, when the request from the JS agent was made. We recommend to
```

Could have been a mobile agent too, right?

```suggestion
      Time expressed according to ISO 8601 in UTC format, when the request from the client agent was made. We recommend to
```